test/v2/controllers/V2ArrivalsControllerSpec.scala

```diff
@@ -675,4 +682,80 @@ class V2ArrivalsControllerSpec
     }
   }
 
+  "GET  /traders/:EORI/movements/arrivals" - {
+    "should return ok with json body for arrivals" in {
+
```

Comment:
we can use forAll(Gen.listOfN(arbitrary[MovementResponse]) something like that with this?

Reply:
it should be fine, we can keep the list as is in seq[], can't see adding much value to this